src/plugins/home/public/application/components/add_data/add_data.tsx

```diff
@@ -138,18 +139,62 @@ export const AddData: FC<Props> = ({ addBasePath, application, isDarkMode, isClo
             </EuiFlexGroup>
           </EuiFlexItem>
 
-          <EuiFlexItem>
-            <EuiImage
-              alt="Illustration of Elastic data integrations"
-              className="homDataAdd__illustration"
-              src={
-                addBasePath('/plugins/kibanaReact/assets/') +
-                (isDarkMode
-                  ? 'illustration_integrations_darkmode.svg'
-                  : 'illustration_integrations_lightmode.svg')
-              }
-            />
-          </EuiFlexItem>
+          {isCloudEnabled && (
+            <EuiFlexItem>
+              <EuiPanel paddingSize="l">
+                <EuiFlexGroup className="moveYourData__panel" alignItems="center" gutterSize="xl">
+                  <EuiFlexItem>
+                    <EuiImage
+                      alt="Illustration of Elastic data integrations"
+                      className="moveYourData__illustration"
+                      src={
+                        addBasePath('/plugins/kibanaReact/assets/') +
+                        (isDarkMode
+                          ? 'illustration-cloud-migration.svg'
```

Comment:
~It looks like the same svg is being used here?~ Based on discussions offline, I think you're still looking into this